### DISCUSSION TITLE
Can not place anything into div

### DISCUSSION THREAD

**Discussion by insssomniac** [GrapesJS/grapesjs#4461] [Q&A]
Hello!

I have a problem with some custom blocks -- I can't place anything inside the divs. 

Let it be for example very simple component like basic 1 column block with Tailwind styling:
```
<div class="p-2 flex">
  <div class="min-h-[75px] w-full">
  
  </div>
</div>
```
When I adding this component to working area, I can't put anything _inside_ the div element, only next to it:
![image](https://user-images.githubusercontent.com/42058438/179952795-730c96ac-4d2b-4185-b317-847fc42c24b5.png)
![image](https://user-images.githubusercontent.com/42058438/179953009-371dfc94-f721-4fe3-bbc5-8f06c808c508.png)

**Comment by insssomniac** [comment]
well, it looks like what i have some white spaces inside div

**Comment by ronaldohoch** [comment]
Can you provide some code so we can check it?
By your first print, you can't drop anything inside text elements in grapesjs

https://github.com/artf/grapesjs/blob/dev/src/dom_components/model/ComponentText.js#L8
